src/data/countries.js

```diff
@@ -13,75 +13,249 @@ class CountryData {
-	new CountryData("Malaysia","Malaysian","MY","MYS"),
-	new CountryData("Mauritius","Mauritian","MU","MUS"),
-	new CountryData("Mexico","Mexican","MX","MEX"),
-	new CountryData("Monaco","Monegasque","MC","MCO"),
-	new CountryData("Montenegro","Montenegrin","ME","MNE"),
-	new CountryData("Nepal","Nepalese","NP","NPL"),
-	new CountryData("Netherlands","Dutch","NL","NLD"),
-	new CountryData("New Zealand","New Zealander","NZ","NZL"),
-	new CountryData("Norway","Norwegian","NO","NOR"),
-	new CountryData("Panama","Panamanian","PA","PAN"),
-	new CountryData("Philippines","Filipino","PH","PHL"),
-	new CountryData("Poland","Polish","PL","POL"),
-	new CountryData("Portugal","Portuguese","PT","PRT"),
-	new CountryData("Russia","Russian","RU","RUS"),
-	new CountryData("Singapore","Singaporean","SG","SGP",["SG", "Singapura", "Republik Singapura"]),
-	new CountryData("Sri Lanka","Sri Lankan","LK","LKA"),
-	new CountryData("Slovakia","Slovak","SK","SVK"),
-	new CountryData("Slovenia","Slovenian","SI","SVN"),
-	new CountryData("South Africa","South African","ZA","ZAF"),
-	new CountryData("South Korea","South Korean","KR","KOR",["Republic of Korea"]),
-	new CountryData("Spain","Spanish","ES","ESP"),
-	new CountryData("Sweden","Swedish","SE","SWE"),
-	new CountryData("Switzerland","Swiss","CH","CHE"),
-	new CountryData("Thailand","Thai","TH","THA"),
-	new CountryData("Turkey","Turkish","TR","TUR"),
-	new CountryData("Ukraine","Ukrainian","UA","UKR"),
-	new CountryData("United Kingdom", "British","GB","GBR", ["UK"]),
-	new CountryData("United States","American","US","USA", ["USA"]),
-	new CountryData("United Arab Emirates","Emirati","AE","ARE",["AE", "UAE"]),
-	new CountryData("Uruguay","Uruguayan","UY","URY"),
-	new CountryData("Venezuela","Venezuelan","VE","VEN"),
-	new CountryData("Vietnam","Vietnamese","VN","VNM", ["Viet Nam","Republic of Viet Nam","South Vietnam"]),
-	new CountryData("Zimbabwe","Zimbabwean","ZW","ZWE")
+	new CountryData("Australia", "Australian", "AU", "AUS"),
+	new CountryData("Austria", "Austrian", "AT", "AUT"),
+	new CountryData("Azerbaijan", "Azerbaijani", "AZ", "AZE"),
+	new CountryData("Bahamas", "Bahamian", "BS", "BHS"),
+	new CountryData("Bahrain", "Bahraini", "BH", "BHR"),
+	new CountryData("Bangladesh", "Bengali", "BD", "BGD"),
+	new CountryData("Barbados", "Barbadian", "BB", "BRB"),
+	new CountryData("Belarus", "Belarusian", "BY", "BLR"),
+	new CountryData("Belgium", "Belgian", "BE", "BEL"),
+	new CountryData("Belize", "Belizean", "BZ", "BLZ"),
+	new CountryData("Benin", "Beninese", "BJ", "BEN"),
+	new CountryData("Bermuda", "Bermudian", "BM", "BMU"),
+	new CountryData("Bhutan", "Bhutanese", "BT", "BTN"),
+	new CountryData("Bolivia", "Bolivian", "BO", "BOL"),
+	new CountryData("Bonaire, Sint Eustatius and Saba", "Dutch", "BQ", "BES"),
+	new CountryData("Bosnia and Herzegovina", "Bosnian", "BA", "BIH"),
+	new CountryData("Botswana", "Batswana", "BW", "BWA"),
+	new CountryData("Brazil", "Brazilian", "BR", "BRA"),
```

Comment:
This one and a few more flags URLs below are broken. To see that you can run `npm run update-docs` and review the updated readme.md file.